CHANGELOG.md

```diff
@@ -4,19 +4,85 @@ All notable changes to this project will be documented in this file.
+
+- Set PyTorch Lightning 1.0 as the minimum requirement ([#274](https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/274))
+
+- Move `pl_bolts.callbacks.self_supervised.BYOLMAWeightUpdate` to  `pl_bolts.callbacks.byol_updates.BYOLMAWeightUpdate` ([#288](https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/288))
+
+- Move `pl_bolts.callbacks.self_supervised.SSLOnlineEvaluator` to `pl_bolts.callbacks.ssl_online.SSLOnlineEvaluator` ([#288](https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/288))
+
+- Move `pl_bolts.datamodules.*_dataset` to `pl_bolts.datasets.*_dataset` ([#275](https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/275))
+
 ### Fixed
 
+- Fixed duplicate warnings when optional packages are unavailable ([#341](https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/341))
+
+- Fixed ModuleNotFoundError when importing datamoules ([#303](https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/303))
+
+- Fixed cyclic imports in `pl_bolts.utils.self_suprvised` ([#350](https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/350))
+
+- Fixed VAE loss to use KL term of ELBO ([#330](https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/330))
+
+- Fixed dataloders of `MNISTDataModule` to use `self.batch_size` ([#331](https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/331))
+
+- Fixed missing `outputs` in SSL hooks for PyTorch Lightning 1.0 ([#277](https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/277))
+
 ### Removed
 
 ### Deprecated
 
-## [0.1.0] - yyyy-mm-dd
+## [0.2.5] - 2020-10-12
+
+### Added
+
+- Enabled PyTorch Lightning 1.0 compatibility
+
+## [0.2.4] - 2020-10-12
+
+## [0.2.3] - 2020-10-12
+
+### Added
+
+- Enabled PyTorch Lightning 0.10 compatibility ([#264]())
+- Added dummy datasets ([#266](https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/266))
+- Added semantic segmentation model `SemSegment` with `UNet` backend ([#259](https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/259))
+- Added `KittiDataModule` ([#248](https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/248))
+- Added `UNet` ([#247](https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/247))
+- Added reinforcement learning models, losses and datamodules ([#257](https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/257))
+
+## [0.2.2] - 2020-09-14
+
+## [0.2.1] - 2020-09-13
+
+### Added
+
+- Added pretrained VAE with resnet encoders and decoders
+- Added pretrained AE with resnet encoders and decoders
+- Added CPC pretrained on CIFAR10 and STL10
+- Verified BYOL implementation
+
+### Changed
+
+- Dropped all dependencies except PyTorch Lightning and PyTorch
+
+## [0.2.0] - 2020-09-10
+
+## [0.1.1] - 2020-08-23
+
+## [0.1.0] - 2020-07-02
```

Comment:
If you open your git IDE you should see releases as tags and then you can easily check what commits were merged to master in between... may you pls copy/paste a few, if you are not sure what kind of add/fix/remove it was you can skip these subsections but it would be nice to cover the main ones 🐰

Reply:
Thank you for the guide :] I have added some more to the changelog.